components/concept/engine/src/main/java/mozilla/components/concept/engine/request/RequestInterceptor.kt

```diff
@@ -72,6 +72,7 @@ interface RequestInterceptor {
     fun onLoadRequest(
         engineSession: EngineSession,
         uri: String,
+        lastOnLoadRequestUri: String?,
```

Comment:
Let's give this a better name here and API docs! Maybe just `lastUri` or `previousUri`?

Reply:
While we're at it the API docs could still need some cleanup. :).

nit: `uri The the` (double the)
nit: ` hasUserGesture` If the request *is* triggered by...
nit: `isRedirect` If the request is due to *a* redirect then true, else false....

Reply:
Sure, I'll update with `lastUri`